internal/cluster/transition.go

```diff
@@ -75,7 +75,7 @@ func (th *transitionHandler) PostResetCluster(sw stateswitch.StateSwitch, args s
 	}
 
 	return th.updateTransitionCluster(logutil.FromContext(params.ctx, th.log), params.db, sCluster,
-		params.reason)
+		params.reason, "LogsCollectedAt", strfmt.DateTime(time.Time{}))
```

Comment:
U use wrong db column here
controller_logs_collected_at

Reply:
You are right, updating.